src/config.sh

```diff
@@ -187,20 +191,23 @@ function set_config_value()
   local option="$1"
   local value="$2"
   local path="$3"
+  local flag="$4"
+
+  flag=${flag:-'SILENT'}
 
   path=${path:-"${PWD}/${KW_DIR}/${name}"}
 
   # The 's' option is usually followed by /, however, this convention will not
   # work well if we deal with paths. Here we had to break the pattern a little
   # bit and use < instead of / after the s option to ensure that we accept
   # paths in the config option.$
-  sed -i -r "s<($option=).*<\1$value<" "$path"
-  sed -i -r "s<#\s*$option<$option<" "$path"
+  cmd_manager "$flag" "sed -i -r \"s<(${option}=).*<\1${value}<\" \"${path}\""
+  cmd_manager "$flag" "sed -i -r \"s<\#\s*${option}<${option}<\" \"${path}\""
```

Comment:
I merged a slight change in this area of the code; please rebase your branch against the unstable branch.